tests/foreman/api/test_computeresource_gce.py

```diff
@@ -164,6 +164,7 @@ def gce_hostgroup_puppet(
 class TestGCEComputeResourceTestCases:
     """Tests for ``api/v2/compute_resources``."""
 
+    @pytest.mark.e2e
```

Comment:
Same here as I mentioned in my first statement, It is not an ideal end-to-end test. This is just a CRUD test.

Reply:
Updated.

Reply:
remove marker from this test